services/ansible/ansible-collection-downloads.service.js

```diff
@@ -0,0 +1,50 @@
+import Joi from 'joi'
+import { renderDownloadsBadge } from '../downloads.js'
+import { BaseJsonService, pathParams } from '../index.js'
+
+const ansibleCollectionSchema = Joi.object({
```

Comment:
We can completely ditch the fields we are not using. So the schema for downloads just needs to be

```js
const ansibleCollectionSchema = Joi.object({
  download_count: Joi.number().required(),
}).required()
```

and for version

```js
Joi.object({
  highest_version: Joi.object({
    version: Joi.string().required(),
  }).required(),
}).required()
```

Reply:
Gotcha, apologies for the misunderstanding. Thanks for working through these PRs with me!